.github/workflows/web-api.yml

```diff
@@ -45,13 +45,19 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
maybe remove. don't think its needed

Reply:
it's needed to get the full commit list (revisions + messages)

Reply:
actually, solved by getting exact number of commits